birdhouse/optional-components/all-public-access/all-public-access-magpie-permission.cfg

```diff
@@ -0,0 +1,235 @@
+permissions:
+  - service: catalog
+    permission: getcapabilities
+    user: authtest
```

Comment:
You don't have to repeat the exact same authorizations as group `anonymous` for user `authtest`, he's part of that group anyways so it's redundant.  

However you need to add read permissions for folder https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/catalog/birdhouse/testdata/secure/catalog.html that group `anonymous` should not have.

I know the current version of Magpie do not support removing read permissions if  the parent already has it.  The next Magpie should be able to do this.

Reply:
The `group:anonymous` permissions are removed, so it won't be redundant.
And for the read access to thredds, it's included in the permissions (see latest block).

```
 - service: thredds
    permission: read
    user: authtest
    action: create
```

Reply:
oh no, you should keep group `anonymous` and remove user `authtest`, except when you give it extra perm to read the `secure` folder.

All the testsuite use anonymous access !

Reply:
Oh alright, so i'll set all the wps permissions for anonymous, and keep the latest block about authtest and thredds.